docs/pages/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.mdx

```diff
@@ -0,0 +1,306 @@
+created through the cluster.
+
+This deployment architecture isn’t recommended for use cases where your users or resources are 
+clustered in a single region or for Managed Service Providers needing to provide separate clusters 
+to customers. Additionally, this architecture is not a solution for increasing the scalability of 
+a single cluster.
+
+We recommend this for globally distributed resources and end-users that prefer a single point of
+entry while also ensuring minimal latency when accessing connected resources.
+
+### Key deployment components
+- High Availability Teleport Cluster
+- Auth Servers must remain in a single region
+- Proxies are deployed across multiple regions
+- [AWS Route 53 latency based routing]([Latency-based routing](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-policy-latency.html) 
+       to create [GSLB](https://www.cloudflare.com/learning/cdn/glossary/global-server-load-balancing-gslb/)
+- [Teleport TLS Routing](https://goteleport.com/docs/architecture/tls-routing/) to reduce the number of ports needed to use Teleport
+- [Teleport Proxy Peering](https://goteleport.com/docs/architecture/proxy-peering/) for reducing the number of resource connections
+- [AWS Network Load Balancing](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html)
+- [AWS DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Introduction.html) for cluster state storage
+- [AWS S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) for session recording storage
+
+## Advantages of this deployment architecture
+- Eliminates the complexity maintaining multiple Teleport clusters across multiple regions
+- Uses the lowest latency path to connect users to resources
+- Provides a highly-resilient, redundant HA architecture for Teleport that can quickly 
+  scale with an organization’s needs.
+- All required Teleport components can be provisioned within the AWS ecosystem.
+- Using load balancers for the Proxy and Auth services allows for increased availability 
+  during Teleport Cluster upgrades. Instances can easily be removed and added while 
+  limiting impact to active users.
+
+## Disadvantages of this deployment architecture
+- When Teleport Auth servers are limited to a single region, there is a higher likelihood 
+  of decreased availability during an AWS regional outage.
+- Technically complex to deploy
+- Long-term cost may be a prohibitive factor for some organizations and can increase total 
+  cost of ownership (TCO) throughout the system’s lifetime cycle.
+
+
+![Diagram of a high-availability Teleport
+architecture](../../img/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.png)
+
+
+## AWS Network load balancer(NLB)
+For this deployment architecture we recommend using AWS NLBs if you plan 
+to use Teleport TLS routing and Proxy Peering. The NLB forwards traffic 
```

Comment:
We already listed that proxy peering is a key component of this architecture, so are NLBs a recommendation or required?

I think people looking for opinionated architectures like this want you to be very specific.

Reply:
"AWS NLBs are required for this highly available deployment architecture. 
The NLB forwards traffic from users and services to an available Teleport instance. This must not 
terminate TLS, and must transparently forward the TCP traffic it receives. 
In other words, this must be a Layer 4 load balancer, not a Layer 7 
(e.g., HTTP) load balancer. "

The NLBs aren't required for either of those components, so I removed the mention and pointed out that the NLBs are required for high availability in this deployment.